internal/api/configuration_profile.go

```diff
@@ -94,38 +94,62 @@ type JiraProject struct {
 
 // ResourceOwnerConfiguration is the configuation for resource owner.
 type ResourceOwnerConfiguration struct {
-	Default      []string `graphql:"resourceOwnerDefault: default"`
-	OrgDomain    *string
-	OrgDomainTag *string
-	Tags         []string
+	Default      []string `graphql:"resourceOwnerDefault: default" json:"default"`
```

Comment:
given that this was a surprise to me, perhaps a comment about the colliding `default`s would be helpful down the line?